src/content/developers/tutorials/run-node-raspberry-pi/index.md

```diff
@@ -92,57 +99,70 @@ User: ethereum
 
-```shell
-sudo systemctl start geth-lh
-sudo systemctl start lh-geth-beacon
+```sh
+sudo journalctl -u geth -f
 ```
 
-The logs can be checked using
+The consensus client does need to be started explicitly. To do this, first open port 9000 on your router so that Lighthouse can find and connect to peers. Then enable and start the lighthouse service:
 
-```shell
-# logs for Geth
-sudo journalctl -u geth-lh -f
-#logs for lighthouse
-sudo journalctl -u lh-geth-beacon -f
+```sh
+sudo systemctl enable lighthouse-beacon
+sudo systemctl start lighthouse-beacon
+```
+
+Check the client using the logs:
+
+```sh
+sudo journalctl -u lighthouse-beacon
 ```
 
-The specific service names for every combination of clients are available at the [Ethereum on Arm docs](https://ethereum-on-arm-documentation.readthedocs.io/en/latest/kiln/kiln-testnet.html#id2). They can be used to update the commands provided here for any combination.
+Note that the consensus client will sync in a few minutes because it uses checkpoint sync. The execution client will take longer - potentially several hours, and it will not start until the consensus client is already finished syncing (this is because the execution cleint needs a target to sync to, which is provided by the synced consensus client).
+
+With the Geth and Lighthouse services running and synced, your Raspberry Pi is now an Ethereum node! It is most common to interact with the Ethereum network using Geth's Javascript console which can be attached to the Geth client on port 8545. It is also possible to submit commands formatted as JSON objects using a request tool such as Curl. See more in the [Geth documentation](geth.ethereum.org).
+
+Geth is preconfigured to report metrics to a Grafana dashboard which can be viewed in the browser. More advanced users might wish to use this feature to monitor the health of their node by navigating to `ipaddress:3000` passing `user: admin` and `passwd: ethereum`.
 
 ## Validators {#validators}
 
-In order to run a validator you must first have access to 32 testnet ETH, which must be deposited into the Kiln deposit contract. This can be done by following the step-by-step guide on the [Kiln Launchpad](https://kiln.launchpad.ethereum.org/en/). Do this on a desktop/laptop, but do not generate keys—this can be done directly on the Raspberry Pi.
+A validator can also be optionally added to the consensus client. The validator software allows yoru node to participate actively in consensus and providing the network with cryuptoeconomic security. You get rewarded for this work in ETH. In order to run a validator you must first have 32 ETH, which must be deposited into the deposit contract. **This is a long term commitment - it is not yet possible to withdraw this ETH!**. The deposit can be made by following the step-by-step guide on the [Launchpad](https://launchpad.ethereum.org/). Do this on a desktop/laptop, but do not generate keys — this can be done directly on the Raspberry Pi.
```

Comment:
```suggestion
A validator can also be optionally added to the consensus client. The validator software allows your node to participate actively in consensus and provides the network with cryptoeconomic security. You get rewarded for this work in ETH. To run a validator, you must first have 32 ETH, which must be deposited into the deposit contract. **This is a long-term commitment - it is not yet possible to withdraw this ETH!**. The deposit can be made by following the step-by-step guide on the [Launchpad](https://launchpad.ethereum.org/). Do this on a desktop/laptop, but do not generate keys — this can be done directly on the Raspberry Pi.
```